### ISSUE TITLE
Try to add ChatGPT API to Jarvis...will me it more lively

### ISSUE THREAD

**Issue by Zaidtech** [sukeesh/Jarvis#1086]

**Comment by jhutchings99** [comment]
Hey @Zaidtech! Are you working on implementing this issue? If not I would love to work on it, please let me know!

**Comment by jomama11** [comment]
this seems interesting, but doesn't open API cost money if you wanted to use it?

**Comment by Zaidtech** [comment]
Yeah @jomama11, but was thinking of a way to scrap and use it...if you have any idea how can we approach that do tell me ...will be working together on it

**Comment by Zaidtech** [comment]
> Hey @Zaidtech! Are you working on implementing this issue? If not I would love to work on it, please let me know!

Yeah you can work ....provided you can pay for the API!

**Comment by jomama11** [comment]
Hey @Zaidtech I don't think I could afford openAI keys for now, even if they do provide a free 5$ credit. But, if anyone could add a feature for any user to pay some kind of subscription for Jarvis that includes chat-gpt, that could be a different way to add chat-gpt to Jarvis.

**Comment by pnhofmann** [comment]
Hmm, chatgpt seems like a bit of an overkill.

But GPT-2 is actually Open Source. Not nearly as powerful as GPT-3 or GPT-4, but still really good. Alternatively there is StableLM. Or some other alternatives.

If someone wants to look into any of that (I haven't) - maybe, could be actually nice if we could use anything for better recognition of characters or make answers more natural.

**Comment by jomama11** [comment]
@pnhofmann it could have a free version with gpt-2 included and also a subscription option with either gpt-3, gpt-3.5 or gpt-4.

**Comment by jhutchings99** [comment]
I got the plugin created and working using gpt-2! Just posted my merge request

**Comment by Her0Class08** [comment]
Could it be made by having the individual user create their own API?
